pkg/img/layer.go

```diff
@@ -10,8 +10,10 @@ import (
 type LayerName string
 
 const (
-	ConfigLayer LayerName = "config"
-	CRDsLayer   LayerName = "crds"
+	ConfigLayer      LayerName = "config"
+	CRDsLayer        LayerName = "crds"
+	DefaultCRLayer   LayerName = "default-cr"
+	RawManifestLayer LayerName = "raw-manifest"
```

Comment:
this one should relocated to `api` module, and this rawmanifestlayer is already defined as `v1beta2.RawManifestLayerName`